client/src/templates/Challenges/classic/classic.css

```diff
@@ -17,53 +17,50 @@
   display: none !important;
 }
 
-.action-row {
+.monaco-editor-tabs {
   padding: 10px;
   border-bottom: 1px solid var(--quaternary-background);
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(8rem, 1fr));
```

Comment:
This one is responsible for responsive design for the buttons `index.html`, `styles.css`, `preview` and console, in both mobile and desktop layout.

Unless I messed up, in math there shouldn't be any difference in the responsive design

Reply:
@Sboonny Thank you for commenting on your pr. It is always appreciated.